packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -5760,6 +5760,48 @@ QUnit.module('datebox validation', {}, () => {
     });
 });
 
+[
+    { type: 'time', input: 'p', displayFormat: 'HH:mm a', expectedValue: 'PM' },
+    { type: 'time', input: 'a', displayFormat: 'HH:mm a', expectedValue: 'AM' },
+    { type: 'time', input: 'x', displayFormat: 'HH:mm a', expectedValue: null },
+    { type: 'time', input: 'y', displayFormat: 'HH:mm a', expectedValue: null },
+    { type: 'time', input: 'z', displayFormat: 'HH:mm a', expectedValue: null },
+    { type: 'datetime', input: 'p', displayFormat: 'dd/MM/yyyy HH:mm a', expectedValue: 'PM' },
```

Comment:
why do you use displayFormat parameter here? in the beginning of your test you set it to 'a' anyway.             this.instance.option('displayFormat', 'a');